src/jit/lclvars.cpp

```diff
@@ -239,9 +239,11 @@ void Compiler::lvaInitTypeRef()
     // Finally the local variables
     //-------------------------------------------------------------------------
 
-    unsigned                varNum    = varDscInfo.varNum;
-    LclVarDsc*              varDsc    = varDscInfo.varDsc;
-    CORINFO_ARG_LIST_HANDLE localsSig = info.compMethodInfo->locals.args;
+    unsigned                argRegNum       = varDscInfo.intRegArgNum;
+    unsigned                otherArgRegNum  = varDscInfo.floatRegArgNum;
```

Comment:
This is confusing at best, and needs a comment. I gather that this handles the case where there may be a mix of int and float regs, so that this is the first of each that is used for this arg?